.github/workflows/scheduled_tests.yaml

```diff
@@ -82,7 +88,7 @@ jobs:
 
       - name: Install dependencies
         run: |
-          python -m pip install --upgrade pip nox
+          python -m pip install --upgrade "pip" "nox[uv]"
```

Comment:
We can probably remove them, since the shell being used is `bash`. I have started including them because my shell is `zsh` and it handles pattern matching differently. Would you want me to remove them? i.e., `pip install --upgrade pip nox[uv]` and `pip install --upgrade "pip" "nox[uv]"` mean the same thing.

Reply:
Removed them, should be better!